envoy/api/v2/cds.proto

```diff
@@ -356,6 +356,18 @@ message Cluster {
     // .. note::
     //   The specified percent will be truncated to the nearest 1%.
     core.Percent healthy_panic_threshold = 1;
+    // [#not-implemented-hide:]
+    // Configures percentage of requests that will be routed to same zone
+    // If not specified, the default is 100%.
+    // :ref:`runtime values <config_cluster_manager_cluster_runtime_zone_routing>`.
+    // :ref:`Zone aware routing support <arch_overview_load_balancing_zone_aware_routing>`.
+    core.Percent zone_aware_routing_enabled = 2;
+    // [#not-implemented-hide:]
+    // Configures minimum upstream cluster size for zone aware routing
```

Comment:
"Configures the minimum upstream cluster size required for zone aware routing. If an upstream cluster has less members than this size, zone aware routing will not be performed, even if configure."

Reply:
Updated . Thanks!